backend/frontend/src/screens/PhoneNumberLoginScreen.js

```diff
@@ -16,6 +16,13 @@ import { loginPhoneNumber, validatePhone } from "../actions/actions";
 
 import "../css/phonenumscreen.css";
 
+const linkStyle = {
+  textDecoration: "underline",
+  color: "black",
+  fontWeight: 500,
+  fontSize: "17px",
+};
+
```

Comment:
do not use inner css

Reply:
removed css from screen and included in css file.